sdk/export/metric/aggregator/aggregator.go

```diff
@@ -63,6 +63,20 @@ type (
 		Points() ([]core.Number, error)
 	}
 
+	// Buckets represents histogram buckets boundaries and counts.
+	//
+	// For a Histogram with N defined boundaries, e.g, [x, y, z].
+	// There are N+1 counts: [-inf, x], (x, y], (y, z], (z, +inf]
```

Comment:
[opencensus metrics](https://github.com/census-instrumentation/opencensus-proto/blob/be218fb6bd674af7519b1850cdf8410d8cbd48e8/src/opencensus/proto/metrics/v1/metrics.proto#L209-L219) has [-inf, x), [x, y), [y, z), [z, +inf). It is a minor difference but could create compatibility issue.

Reply:
That is interesting. I wonder what are the most used operator on histograms, `less than` or `less than or equals`. Prometheus, for example, uses the `less than or equal` operator.  The differences should be little in the end.

I'll leaning to change it to be `less than` to be consistent with OpenCensus, if anyone have another opinion here would be great.

Reply:
It looks like [the decision in open census](https://github.com/census-instrumentation/opencensus-proto/pull/121/commits/6543b80d05e3b2a65c93fbe3eae6679c7451c666#diff-ae7519afabde512d65fb5cb5ff33caa4R210-R220) was motivated by trying to remain compatible with [OpenMetrics](https://github.com/OpenObservability/OpenMetrics/pull/126/files#diff-d1792782ae179ab49b581de250c5def2R195-R203). That being said, it looks like it is still [an open issue](https://github.com/OpenObservability/OpenMetrics/issues/62) in the OpenMetric project.

I still think going with `(-inf, x), [x, y), [y, z), [z, +inf)` is the correct approach, and at this point I don't think it locks us into too much.

Reply:
Thanks everyone for discussing this. I've changed the code to be consistent with the Open Census approach.